app/views/install/compose.phtml

```diff
@@ -633,6 +574,32 @@ services:
       - _APP_REDIS_USER
       - _APP_REDIS_PASS
 
+  openruntimes-executor:
+    container_name: openruntimes-executor
+    hostname: exc1
+    <<: *x-logging
+    stop_signal: SIGINT
+    image: openruntimes/executor:0.1.3
+    networks:
+      - appwrite
+      - openruntimes-runtimes
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - openruntimes-builds:/storage/builds:rw
+      - openruntimes-functions:/storage/functions:rw
+      - /tmp:/tmp:rw
+    environment:
+      - OPR_EXECUTOR_CONNECTION_STORAGE
+      - OPR_EXECUTOR_INACTIVE_TRESHOLD
+      - OPR_EXECUTOR_NETWORK
+      - OPR_EXECUTOR_DOCKER_HUB_USERNAME
+      - OPR_EXECUTOR_DOCKER_HUB_PASSWORD
```

Comment:
Let's substitute these variables as well with `OPR_EXECUTOR_NETWORK=$_APP_EXECUTOR_NETWORK` and the likes

Reply:
Done.